src/main/java/com/hobbycollection/service/ICollectionService.java

```diff
@@ -0,0 +1,11 @@
+package com.hobbycollection.service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public interface ICollectionService {
+    Collection getItemByID(int id);
+    Collection save(Collection collection) throws Exception;
+    ArrayList<Collection> fetchAll();
+    void delete(int id);
+}
```

Comment:
I added a Service Interface ICollectionService.
Services are business logic so they are essential for the application function. I add a Service Interface 
       to include methods.
       "Services make it easier to avoid overly-tight coupling between components, by using explicit remote call mechanisms.
Sources: 
     * Lecture (Videos)
     * https://martinfowler.com/articles/microservices.html